.circleci/config.yml

```diff
@@ -77,7 +77,13 @@ jobs:
               find . -maxdepth 1 -type d -not \( -path . -o -path ./Integrations -o -path ./Scripts -o -path ./Beta_Integrations -o -path ./venv \) | xargs ./Tests/scripts/pyflake.sh
 
               [ -n "${BACKWARD_COMPATIBILITY}" ] && CHECK_BACKWARD=false || CHECK_BACKWARD=true
-              python ./Tests/scripts/validate_files.py -c true -b $CHECK_BACKWARD
+              if [[ $CIRCLE_BRANCH =~ pull/[0-9]+ ]] ;
```

Comment:
i think that it would be better to do this valiidation in the validate_files file, we already extract the branch name there, and we have the external regex that Fruend created few days ago

Reply:
done